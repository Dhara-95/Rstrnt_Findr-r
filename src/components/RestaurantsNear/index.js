import React from "react";

import { LabelsObject } from "../../Labels";

const RestaurantsNear = (props) => {
  const { userInput } = props;
  const { searchResultQueryLabel } = LabelsObject;

  return (
    <h2 className="text-grey">
      {searchResultQueryLabel}
      {userInput}
    </h2>
  );
};

export { RestaurantsNear };
