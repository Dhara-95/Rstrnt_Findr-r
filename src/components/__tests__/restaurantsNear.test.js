import React from "react";
import renderer from "react-test-renderer";

import { RestaurantsNear } from "../RestaurantsNear";

it("displays the Restaurants Near Header correctly", () => {
  const tree = renderer
    .create(<RestaurantsNear userInput={"W13 3RH"} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
