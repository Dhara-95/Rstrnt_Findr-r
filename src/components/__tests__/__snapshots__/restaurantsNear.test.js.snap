// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the Restaurants Near Header correctly 1`] = `
<h2
  className="text-grey"
>
  Restaurants near 
  W13 34H
</h2>
`;
